code/modules/surgery/embalming.dm

```diff
@@ -22,7 +21,7 @@
 /datum/surgery_step/embalming/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	user.visible_message("[user] embalms [target]'s body.", "<span class='notice'>You succeed in embalming [target]'s body.</span>")
 	ADD_TRAIT(target, TRAIT_HUSK, MAGIC_TRAIT) //Husk's prevent body smell
```

Comment:
What the heck, could you add a different new trait instead? This can lead to some peculiarities with limbs icons and other aspects of the game.
Here's the line that needs to be changed once the new trait is added:
https://github.com/Citadel-Station-13/Citadel-Station-13/blob/1c811c3f88c73bd896b1735104ec1e09c9a2c810/code/modules/mob/living/carbon/life.dm#L360
Also I want to make formaldehyde apply such a trait later.

Reply:
This one is different tho, you permitintly husking the person, other one can be removed by splashing flesh on them.
Or does it force remove husked when synthfleshed?

Reply:
Husking is not quite the same as embalming a body. That's why I tell you to use another flag.

Reply:
what trait do you want for it?